tasks/Resources/RHODS_OLM/uninstall/uninstall.robot

```diff
@@ -19,7 +19,16 @@ Uninstall RHODS In OSD
   Trigger RHODS Uninstall 
     
 Uninstall RHODS In PSI
-  Fail  Not implemented yet
+      ${return_code}    ${output}	  Run And Return Rc And Output   git clone ${RHODS_INSTALL_REPO}
+      Log   ${output}
+      Should Be Equal As Integers	${return_code}	 0
+      ${git_folder} =  Get Regexp Matches    ${output}	   Cloning into \'(.*?)\'    1
+      Log   ${git_folder}[0]
+      ${return_code}    ${output}    Run And Return Rc And Output   (cd ${git_folder}[0]; ./cleanup.sh); wait $!; sleep 60   #robocop:disable
```

Comment:
hi @pablofelix   can u add  a log to console the  cmd which will help in debugging before the robot log is been generated  something like
```log to  console cd ${git_folder}[0]; ./cleanup.sh); wait $!; sleep 60```

else LGTM

Reply:
done